src/OpenTelemetry.Api/Trace/ProxyTracer.cs

```diff
@@ -50,51 +50,21 @@ public IDisposable WithSpan(ISpan span, bool endOnDispose)
 
         public ISpan StartRootSpan(string operationName, SpanKind kind, SpanCreationOptions options)
         {
-            if (operationName == null)
```

Comment:
Could these functions also emit logs when values are null?

Reply:
done it on the span impl, so we don't have to repeat it so many times